pkg/types/conversion/installconfig_test.go

```diff
@@ -58,6 +59,7 @@ func TestConvertInstallConfig(t *testing.T) {
 					DeprecatedClusterNetworks: []types.ClusterNetworkEntry{
 						{
 							CIDR: *ipnet.MustParseCIDR("1.2.3.5/32"),
+
```

Comment:
extra line?

Reply:
There is some trouble with gofmt here. the value should be aligned with the other values below, but the gofmt test does not like that. the extra line as seen elsewhere in the file allows this to pass that test, which seems to be incorrectly failing.